ui/pages/asset/components/asset-page.tsx

```diff
@@ -224,28 +247,52 @@ const AssetPage = ({
               <Text variant={TextVariant.headingMd} paddingBottom={4}>
                 {t('tokenDetails')}
               </Text>
-              {renderRow(
-                t('contractAddress'),
-                <AddressCopyButton address={address} shorten />,
-              )}
               <Box
                 display={Display.Flex}
                 flexDirection={FlexDirection.Column}
                 gap={2}
               >
-                {asset.decimals !== undefined &&
-                  renderRow(t('tokenDecimal'), <Text>{asset.decimals}</Text>)}
-                {asset.aggregators && asset.aggregators?.length > 0 && (
+                {type === AssetType.token && (
                   <Box>
-                    <Text
-                      color={TextColor.textAlternative}
-                      variant={TextVariant.bodyMdMedium}
+                    {renderRow(
+                      t('contractAddress'),
+                      <AddressCopyButton address={address} shorten />,
+                    )}
+                    <Box
+                      display={Display.Flex}
+                      flexDirection={FlexDirection.Column}
+                      gap={2}
                     >
-                      {t('tokenList')}
-                    </Text>
-                    <Text>{asset.aggregators?.join(', ')}</Text>
+                      {asset.decimals !== undefined &&
+                        renderRow(
+                          t('tokenDecimal'),
+                          <Text>{asset.decimals}</Text>,
+                        )}
+                      {asset.aggregators && asset.aggregators.length > 0 && (
+                        <Box>
+                          <Text
+                            color={TextColor.textAlternative}
+                            variant={TextVariant.bodyMdMedium}
+                          >
+                            {t('tokenList')}
+                          </Text>
+                          <Text>{asset.aggregators.join(', ')}</Text>
+                        </Box>
+                      )}
+                    </Box>
                   </Box>
                 )}
+                {renderRow(
+                  t('spendingCaps'),
+                  <ButtonLink
+                    className="asset-page__spending-caps mm-text--body-md-medium"
+                    href={portfolioSpendingCapsUrl}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {t('editInPortfolio')}
+                  </ButtonLink>,
```

Comment:
Updating to `ButtonLink` here to adhere to the design system which has a slightly different border for the link interaction.

### Before

https://github.com/user-attachments/assets/2aec75ee-47f1-4d9a-b747-369027961ecf

### After

https://github.com/user-attachments/assets/8c7e462a-ac71-4b8f-8703-2987b865f82b

Still renders semantic HTML as the as is handled by the `href` prop

<img width="682" alt="Screenshot 2024-11-19 at 8 46 37 AM" src="https://github.com/user-attachments/assets/1c0a7d17-45bd-4daf-8e1a-4f3df96a1263">